catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -344,9 +354,19 @@ private Filter buildAttributeFilter(
     return properties;
   }
 
+  private List<MetacardType> getFilteredMetacardTypes() {
+    if (!whitelistedMetacardTypes.isEmpty()) {
```

Comment:
✏️ You don't need this check because if `whitelistedMetacardTypes` is empty then there will be nothing to filter out and it will ultimately return the same thing

Reply:
Nice catch @Schachte. Didn't notice that the stream and the empty check used different collections

Reply:
This isn't the desired behavior. We want to only return filtered if there are whitelisted items, otherwise return all.